assets/dev/scss/frontend/_container.scss

```diff
@@ -157,17 +156,17 @@
 	}
 
 	@mixin border-radius-for-background-elements {
-		border-radius: var(--border-radius);
-		border-style: var(--border-style);
-		border-color: var(--border-color);
-		border-top-width: var(--border-width-top);
-		border-right-width: var(--border-width-right);
-		border-bottom-width: var(--border-width-bottom);
-		border-left-width: var(--border-width-left);
-		top: calc( 0px - var(--border-width-top) );
-		left: calc( 0px - var(--border-width-left) );
-		width: max( calc( 100% + var(--border-width-right) + var(--border-width-left) ), 100% );
-		height: max( calc( 100% + var(--border-width-top) + var(--border-width-bottom) ), 100% );
+		border-radius: var( --border-radius );
+		border-style: var( --border-style );
+		border-color: var( --border-color );
+		border-block-start-width: var( --border-width-top, var( --border-block-start-width ) );
```

Comment:
I don't think that we to use a fallback value here. I don't think the `--border-width-*` are used anywhere in the nested widgets.